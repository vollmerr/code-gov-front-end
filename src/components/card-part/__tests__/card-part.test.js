import React from 'react';

import CardPart from 'components/card-part';
import { shallow } from 'enzyme';

describe('Testing <CardPart /> component', () => {
  const props = {
    title: 'test-title',
    text: 'test-text'
  };
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CardPart {...props} />);
  });
  it('renders 1 CardPart', () => {
    expect(wrapper.find('Fragment').exists()).toEqual(true);
  });
  it('renders 1 CardPart without props', () => {
    wrapper = shallow(<CardPart />);
    expect(wrapper.find('Fragment').exists()).toEqual(true);
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})