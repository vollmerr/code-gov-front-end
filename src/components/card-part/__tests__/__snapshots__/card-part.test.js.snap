// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <CardPart /> component should render correct 1`] = `
<Fragment>
  <dt>
    test-title:
  </dt>
  <dd>
    test-text
  </dd>
</Fragment>
`;
